contracts/blade/ChildERC1155Predicate.sol

```diff
@@ -211,7 +214,10 @@ contract ChildERC1155Predicate is IChildERC1155Predicate, Predicate, Initializab
     ) private onlyValidToken(childToken) {
```

Comment:
Should we then rename the `childToken` to `destinationToken`? Let's have the consolidated terminologies IMO.